backend/qemu.pm

```diff
@@ -876,7 +876,7 @@ sub start_qemu ($self) {
             }
         }
 
-        sp('enable-kvm') unless $vars->{QEMU_NO_KVM};
+        sp('enable-kvm') if -r '/dev/kvm' && !$vars->{QEMU_NO_KVM};
```

Comment:
So what happens if there's no `/dev/kvm` device? It gets switched off silently?

Reply:
Yes

Reply:
I actually moved this change out to https://github.com/os-autoinst/os-autoinst/pull/1889